app/submitter/submitter.py

```diff
@@ -149,7 +160,7 @@ def upload(self, metadata, payload):
 
 class LogFeedbackSubmitter:
     @staticmethod
-    def upload(metadata, payload):
+    def upload(metadata: Mapping[str, Union[str, int, list]], payload: dict) -> bool:
```

Comment:
Consider making MetadataType as constant with `MetadataType=Mapping[str, Union[str, int, list]]` since it i used few times in code.

Reply:
It's only used once once now so I won't set a constant, but would do otherwise, thanks!

Reply:
The type for the metadata should match between the GCS/Log submitter.

Reply:
So I made them a const as @parimooa suggested and re-use it now in both submitters, thanks!